src/lib/storage/segment_iterables/segment_positions.hpp

```diff
@@ -55,12 +55,13 @@ class SegmentPosition : public AbstractSegmentPosition<T> {
 
   const T& value() const final { return _value; }
   bool is_null() const final { return _null_value; }
-  const ChunkOffset& chunk_offset() const final { return _chunk_offset; }
+  ChunkOffset chunk_offset() const final { return _chunk_offset; }
 
  private:
-  const T _value;
-  const bool _null_value;
-  const ChunkOffset _chunk_offset;
+  // The alignment improves the suitability of the iterator for (auto-)vectorization
+  alignas(8) const T _value;
```

Comment:
Do you think that experiments with larger alignment sizes could yield performance improvements?

Reply:
Didn't make a difference.